core/src/main/java/org/mule/routing/filters/ExpressionFilter.java

```diff
@@ -103,18 +101,7 @@ public boolean accept(MuleMessage message)
             Thread.currentThread().setContextClassLoader(expressionEvaluationClassLoader);
 
             // TODO MULE-9341 Remove Filters. Expression filter will be replaced by something that uses MuleEvent.
```

Comment:
Yes, because this method still needs to be supports it's implementing is creating an event to wrap message.